b2luigi/batch/processes/gbasf2.py

```diff
@@ -946,10 +951,11 @@ def setup_dirac_proxy():
     Runs ``gb2_proxy_init -g belle`` if there's no active dirac proxy. If there is, do nothing.
     """
     # first run script to check if proxy is already alive or needs to be initalized
-    if get_proxy_info().get("secondsLeft", 0) > 0:
+    if get_proxy_info().get("secondsLeft", 0) > 3600 * get_setting("gbasf2_min_proxy_lifetime", default=0):
```

Comment:
This got me thinking, we could allow for floating point lifetimes and extract minutes as well, e.g.
```suggestion
    lifetime_setting = get_setting("gbasf2_proxy_lifetime", default=24)
    hours = int(lifetime_setting)
    minutes = int(float(lifetime_setting) % 1 * 60)
```

And later set
``` python
run_with_gbasf2(shlex.split(f"gb2_proxy_init -g belle -v {hours}:{minutes}"), ensure_proxy_initialized=False)
```
If we do so, that should also be documented.  But not sure if it's worth adding the complexity for that. rarely anyone needs minute resolution for the proxy lifetime. However, if we don't do that, I would suggest a warning or exception of the lifetime is not integer.

Reply:
I don't really see much use of having minute-level resolution. The use case I have in mind is setting the lifetime for a few days so it doesn't timeout during monitoring/downloading.

A warning for non-integer input makes sense to me.